src/screens/ContactScreen/index.js

```diff
@@ -115,7 +116,7 @@ const ContactScreen = ({navigation}) => {
           image
         }
       };
-      createSignChat(followed, dataSelected);
+      createSignChat(followed, dataSelected, 'CONTACT_SCREEN');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The error handling here is not sufficient. It's a good practice to handle errors in a more user-friendly way rather than just logging them to the console. Consider showing an error message to the user or retrying the operation if it makes sense in this context.
```diff
     } catch (e) {
-      console.log(e, 'error signed chat');
+      // Show an error message to the user or handle the error appropriately
+      handleError(e);
     }
```
Where `handleError` is a function that handles the error in a way that is appropriate for your application. This could involve displaying an error message to the user, sending the error to an error tracking service, or even retrying the operation if it's likely that it might succeed on a subsequent attempt.


<!-- This is an auto-generated comment by OSS CodeRabbit -->